charter.md

```diff
@@ -53,15 +53,130 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Elections
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+SIG Contributor Experience may delegate this responsibility to a subproject.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this fails to fill the seat, a special election for that position will 
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing. As a courtesy,
+such notifications should be given at least 30 calendar days in advance of
+their departure.
+
+The committee can choose to accept a member's resignation and carry out
+offboarding procedures at any time following the written notice.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by a two-thirds vote of no
+confidence from other Steering Committee members.
+
+The call for a vote will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
+another.
+
+There is a one week community feedback period for contributors to use to reach
+out to Steering Committee members with their thoughts or log them on the open
+issue including non-binding votes. After that period, Steering Committee
```

Comment:
`reach out to Steering Committee members with their thoughts or log them on the open issue including non-binding votes` mixes up a few things, lets separate them out?

`reach out to Steering Committee members individually or collectively using the usual mechanisms (steering public or private mailing lists, steering slack etc). They can also log their thoughts in the github issue (non-binding votes are welcome!).

Reply:
I would discourage members from publishing non-binding votes in advance because it makes it harder for them to change their minds and will create the impression that the board was not sensitive to feedback recieved.

Reply:
I'm concerned about the possibility of "brigading" on a particular impacted person if we encourage public comment on a sensitive issue like removing someone from a leadership position.

It's tricky, because allowing community members to share feedback is critical, and having transparency is important, but for many reasons, we also don't want it to turn into a mob attack either.